openstack/utils/templates/snippets/_proxysql.cfg.tpl

```diff
@@ -15,11 +15,21 @@ mysql_variables =
     {{- $monitorEnabled := false }}
     {{- $monitorUsername := "" }}
     {{- $monitorPassword := "" }}
+    {{- $monitorGalera := false }}
     {{- /*
     Check that proxysql_monitor user exists in all DB instances
     Check that proxysql_monitor credentials are the same in all DB instances
     */}}
     {{- range $dbKey, $db := $dbs }}
+        {{- if hasKey $db "pxc" }}
+            {{- $monitorEnabled = true }}
+            {{- $monitorUsername = "monitor" }}
```

Comment:
Shouldn't the username also come from the config?
I.e. 
```
{{- $monitorUsername = $db.system_users.monitor.user | default "monitor"  }}
```

Reply:
These system users' usernames ('root', 'monitor', 'operator', 'xtrabackup', 'replication' and 'proxyadmin') in pxc-db are fixed and can't be changed, so it always will be a `monitor`, so there would be misleading to use this variable like we can use other user as a monitor user.